1.0/docs/migration.md

```diff
@@ -796,7 +796,7 @@ Example of using the layout classes in the main page:
       <!-- import module -->
       <link rel="import" href="/bower_components/iron-flex-layout/iron-flex-layout-classes.html">
       <!-- include the necessary style modules -->
-      <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
+      <style is="custom-style" include="iron-flex iron-positioning"></style>
```

Comment:
`center-center` is actually defined in both `iron-flex` ([iron-flex-layout-classes.html:70](https://github.com/PolymerElements/iron-flex-layout/blob/e4e1058/iron-flex-layout-classes.html#L70),[77](https://github.com/PolymerElements/iron-flex-layout/blob/e4e1058/iron-flex-layout-classes.html#77)) and `iron-flex-alignment` ([iron-flex-layout-classes.html:152](https://github.com/PolymerElements/iron-flex-layout/blob/e4e1058/iron-flex-layout-classes.html#L152)).